src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -289,25 +289,39 @@ class LicenceConditionService(
     conditionFormatter.format(licencePolicyService.getConfigForCondition(version, conditionCode), data)
 
   @Transactional
-  fun deleteAdditionalConditions(licenceEntity: Licence, conditionIds: List<Long>) {
+  fun deleteConditions(
+    licenceEntity: Licence,
+    additionalConditionIds: List<Long>,
+    standardConditionIds: List<Long>,
+  ) {
     val username = SecurityContextHolder.getContext().authentication.name
     val currentUser = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
       ?: throw RuntimeException("Staff with username $username not found")
 
     // return all conditions except condition with submitted conditionIds
-    val revisedConditions = licenceEntity.additionalConditions.filter { conditionIds.indexOf(it.id) == -1 }
+    val revisedAdditionalConditions =
+      licenceEntity.additionalConditions.filter { additionalConditionIds.indexOf(it.id) == -1 }
```

Comment:
mmh.. would have thought you could have done `contains` here?

Reply:
replaced